core-build/gulp-core-build-sass/package.json

```diff
@@ -23,7 +23,7 @@
     "autoprefixer": "~9.1.3",
     "postcss-modules": "~1.3.1",
     "glob": "~7.0.5",
-    "node-sass": "4.9.3",
+    "node-sass": "4.12.0",
```

Comment:
`node-sass@4.12.0` is required for Node 12 support:

https://github.com/sass/node-sass/releases/tag/v4.12.0

Reply:
Does this introduce a `node-gyp` dependency?  See recent discussion in https://github.com/microsoft/web-build-tools/issues/1361

Reply:
Looks like we're good on that front. [`node-sass@4.9.3`](https://github.com/sass/node-sass/releases/tag/v4.9.3) doesn't support Node 12 natively:

![image](https://user-images.githubusercontent.com/5010588/61175669-4c7db780-a568-11e9-8dcb-44189a04c503.png)

But [`node-sass@4.12.0`](https://github.com/sass/node-sass/releases/tag/v4.12.0) does:

![image](https://user-images.githubusercontent.com/5010588/61175693-8cdd3580-a568-11e9-8b97-027ad9925c63.png)